robojudge.ipynb

```diff
@@ -344,7 +347,7 @@
     "\n",
     "ranking_df = pd.DataFrame()\n",
     "for predictor_name, predictions_file in predictions.items():\n",
-    "    errors = validate_submission(start_date_str, end_date_str, predictions_file)\n",
+    "    errors = validate_submission(start_date_str, end_date_str, IP_FILE, predictions_file)\n",
```

Comment:
The ip file is now passed to `validate_submission` in order to check which countries were requested